cljfmt/src/cljfmt/config.clj

```diff
@@ -68,6 +110,4 @@
    (let [path (if (directory? path)
                 (find-config-file path)
                 path)]
-     (->> (some-> path read-config)
-          (merge default-config)
-          (convert-legacy-keys)))))
+     (load-extra-configs path))))
```

Comment:
Perhaps instead we separate out the merging of extra configs like so:

```clojure
(->> (some-> path read-config)
     (merge default-config)
     (merge-extra-configs)
     (convert-legacy-keys))
```

Reply:
yea I can restructure to do it that way

Reply:
done